src/cljs/main/broadfcui/page/workspace/summary/catalog/questions.cljs

```diff
@@ -131,6 +131,27 @@
                              :onClick #(apply swap! state update :attributes dissoc
                                               (library-utils/get-related-id+label-props library-schema property))})]]))])
 
+(defn- render-populate-typeahead [{:keys [prop property]}]
+  [:div {:style {:marginBottom "0.75em"}}
+   [comps/AutocompleteFilter
+    {:width "100%"
+     :ref "text-filter"
+     :placeholder (:inputHint prop)
```

Comment:
`prop` itself is merged into the data handed to each of these render functions, so you can include it in the map destructure at the top (i.e. `[{:keys [prop property inputHint]}]`) and use it directly here.  Then again, I see that I myself didn't do that below..... so either leave it or change both.